man/os-release.xml

```diff
@@ -218,12 +218,16 @@
 
       <varlistentry>
         <term><varname>HOME_URL=</varname></term>
+        <term><varname>DOC_URL=</varname></term>
```

Comment:
I think "DOCUMENTATION_URL" would be nicer.

Reply:
Especially that it avoids the "doc vs docs" problem.

Reply:
Sure, I can go with that. I'll send an update in a moment.